package.json

```diff
@@ -33,6 +35,7 @@
     "moment": "^2.29.1",
     "mongodb": "^3.6.2",
     "mongoose": "^5.10.9",
+    "node-cron": "^2.0.3",
     "newman": "^5.2.2",
```

Comment:
nit but I believe this list is like partially alphabetized, can you alphabetize it? Will look into prettier taking care of this for us in the future